src/sqlfluff/dialects/dialect_bigquery.py

```diff
@@ -585,18 +585,29 @@ class TypelessStructSegment(BaseSegment):
     type = "typeless_struct"
     match_grammar = Sequence(
         "STRUCT",
-        Bracketed(
-            Delimited(
-                AnyNumberOf(
-                    Sequence(
-                        Ref("BaseExpressionElementGrammar"),
-                        Ref("AliasExpressionSegment", optional=True),
-                    ),
+        Ref("TupleSegment"),
+    )
+
+
+@bigquery_dialect.segment()
+class TupleSegment(BaseSegment):
+    """Expression to construct a TUPLE.
+
+    https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types#tuple_syntax
+    """
+
+    type = "tuple"
+    match_grammar = Bracketed(
```

Comment:
Think this match grammar should just be
```sql
match_grammar = Bracketed(
        Delimited(
            Sequence(
                Ref("BaseExpressionElementGrammar"),
                Ref("AliasExpressionSegment", optional=True),
            ),
        ),
    )
```
that's what's in the docs
![image](https://user-images.githubusercontent.com/80432516/146278815-bcd09ab3-f498-4aa9-b2e7-ffbd9197e377.png)

Reply:
tuple can be named
```
('aaaa' as f1, 'bb' as f2)
```

Reply:
agree, will change

Reply:
will revert `TypelessStruct`
and redefine `Tuple`

Reply:
what was wrong with just changing the match grammar in tuple to what I had in my original comment?

Reply:
Changing TypelessStruct was fine as you originally did it, its just the original match grammar was too verbose.

Reply:
> what about any number of values in tuple?

Delimited contains contains any number of items anyway so the AnyNumber of was redundant

Reply:
when I tried it, it parsed fine for all the existing test cases

Reply:
TypelessStruct and Tuple syntax are partially compatible as of BQ documentation